utam-core/src/main/java/utam/core/selenium/element/ElementAdapter.java

```diff
@@ -109,22 +115,27 @@ protected Element wrapElement(WebElement element) {
 
   @Override
   @SuppressWarnings("rawtypes")
-  public Element findElement(Locator locator) {
-    By by = ((LocatorBy) locator).getValue();
-    WebElement res = getWebElement().findElement(by);
-    if (res == null) { //this can happen for mock in unit tests
-      throw new NoSuchElementException(getNotFoundErr(locator));
+  public Element findElement(Locator locator, boolean isNullable) {
+    // return null for not found element, otherwise throws or returns non empty list
+    List<Element> res = findElements(locator, isNullable);
```

Comment:
It's entirely possible this could break something there. I've not looked at that code in quite some time, so I'd need to debug it with this change and see.

Reply:
if we somehow find it's affected, pls help add unit test :)